Podfile

```diff
@@ -49,7 +49,7 @@ def wordpress_ui
 end
 
 def wordpress_kit
-  pod 'WordPressKit', '~> 8.2.0-beta.1'
+  pod 'WordPressKit', '~> 8.2-beta'
```

Comment:
Should this be a stable version?

Reply:
Hey @oguzkocer, thank you for raising this. This is the version specifier in the `Podfile` and it says "get everything compatible with 8.2, including betas. This way, we can run `bundle exec pod update` and get the latest compatible version without having to change the rule.

At least, in theory. You comment made me realize that I run `install` instead of `update`, resulting in WordPressKit staying at `8.2.0-beta.1`. d503b209e1e0f8160c740d6e87ae8168e9667a9f rectifies this. Thanks!

I've been using this style for a while, but it's either not self explanatory enough or just not a good fit for our current workflow, as folks keep changing the value in the `Podfile` to get what they need, as opposed to run `update`. There's also the chance that a beta version will make it into a release, although I think we have enough review in place to catch that 🤔 

Keen to hear what you think.

Reply:
> it says "get everything compatible with 8.2, including betas

I might be wrong, but I don't think this is what we want. We want to only use stable versions - at least according to the release scenario.

My preference would be to be explicit with our intentions in `Podfile`. However, I don't think it's a big deal either way as long as we are using the correct version.